server/routes/github/github_app.py

```
@@ -0,0 +1,1054 @@
+                    uninstalled_on_github += 1
+                else:
+                    uninstall_failures += 1
+                    logger.warning(
+                        "[GITHUB-DISCONNECT] uninstall returned status=%d installation_id=%d",
+                        resp.status_code, iid,
+                    )
+
+    soft_deleted_installs = 0
+    try:
+        with db_pool.get_admin_connection() as conn:
+            with conn.cursor() as cur:
```

Ruff (0.15.12)

[warning] Use a single `with` statement with multiple contexts instead of nested `with` statements

Combine `with` statements

(SIM117)